Controller/Annotations/Route.php

```diff
@@ -22,9 +22,8 @@ class Route extends BaseRoute
     public function __construct(array $data)
     {
         parent::__construct($data);
-        $requirements = $this->getRequirements();
-        $requirements['_method'] = $this->getMethod();
-        $this->setRequirements($requirements);
+
+        $this->setMethods((array) $this->getMethod());
```

Comment:
It shouldn't be a BC break.
`(array) null` ends up being an empty `[]`, which is is the default value for the methods property, and this code is in the constructor.

Althoug it might be a problem, tested it with both 1.6 and 1.7, and the methods part is definitely overwritten somewhere.